notebooks/xvcfsubsample/main.py

```diff
@@ -0,0 +1,89 @@
+# publish to: "terra-notebook-utils-tests" "test"
+import os
+import herzog
+from firecloud import fiss
+
+with herzog.Cell("markdown"):
+    """
+    # VCF Subsample Workflow Input Preparation
+
+    This notebook demonstrates the creation of Terra data tables as input to xvcfsubsample workflows, referencing input
+    and output VCFs with DRS URIs or Google Storage URLs. It should be used with xvcfsubsample version
+    [0.0.3](https://dockstore.org/workflows/github.com/DataBiosphere/xvcfsubsample/xvcfsubsample:v0.0.3?tab=info)
+
+    ### Workflows
+
+    Import the [Dockstore](https://dockstore.org) workflows into your workspace using the "NHLBI Biodata Catalyst" launch button:
+     - [xvcfsubsample](https://dockstore.org/workflows/github.com/DataBiosphere/xvcfsubsample/xvcfsubsample:v0.0.3?tab=info)
+
+    ### Input File Requirements
+
+    - Input files _must_ be referenced with either DRS URIs or Google bucket URLs.
+    - Output files _must_ be Google bucket URLs. The target bucket will usually be your workspace bucket, but it can be
+      any bucket for which you have write access.
+    """
+
+os.environ['WORKSPACE_NAME'] = "terra-notebook-utils-tests"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-9169fcd1-92ce-4d60-9d2d-d19fd326ff10"
+os.environ['GOOGLE_PROJECT'] = "firecloud-cgl"
+
+with herzog.Cell("python"):
+    #%pip install --upgrade --no-cache-dir terra-notebook-utils
+    pass
+
+with herzog.Cell("python"):
+    import os
+    from terra_notebook_utils import table
+
+    workspace = os.environ['WORKSPACE_NAME']
+    workspace_namespace = os.environ['GOOGLE_PROJECT']
+    workspace_bucket = os.environ['WORKSPACE_BUCKET']
+
+with herzog.Cell("markdown"):
+    """
+    ## Option A: VCF subsample workflow input for DRS URIs
+    This is a typical workflow preparation for merging TOPMed VCFs _without_ downloading them to your workspace.
+    Results will be placed in your workspace bucket.
+    """
+
+table.delete("vcf-subsample-input-drs")
+with herzog.Cell("python"):
+    table_name = "vcf-subsample-input-drs"
+    table.put_row(table_name, dict(workspace=workspace,
+                                   billing_project=workspace_namespace,
+                                   input="drs://dg.4503/b2871873-8dcb-4a3e-a926-a17ab4a19f0a",
+                                   output=f"{workspace_bucket}/subsampled/drs_subsampled_a.vcf.gz",
+                                   samples=["NWD999037", "NWD996859"]))
```

Comment:
Could we provide a commented-out alternative to import samples from a file?

Reply:
Samples may conceivably be stored in several formats such as CSVs, text files, and json. This makes it difficult to create a good example.

However, I've included some language describing how to localize a file to the notebook and open it. Parsing out the content is an exercise left to the reader. Let me know what you think.

Reply:
That's totally fair. I like your approach here!